### PR TITLE
:sparkles: Introduce new config option enableLayout for enabling layout testing.

### PR DESCRIPTION
**Pull request by prklm10**
Adding following to global config
 - enableLayout (default false)

### CODE REVIEW

File: packages/dom/README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ const domSnapshot = await page.evaluate(() => PercyDOM.serialize(options))
 - `enableJavaScript` — When true, does not serialize some DOM elements
 - `domTransformation` — Function to transform the DOM after serialization
 - `disableShadowDOM` — disable shadow DOM capturing, this option can be passed to `percySnapshot` its part of per-snapshot config.
+- `enableLayout` - When true, It enables layout testing.
```
Comment (ninadbstack) [inline packages/dom/README.md:41]:
```suggestion
- `enableLayout` - When true, It enables layout testing on entire screenshot
```

File: packages/core/src/config.js [hunk 2]
```diff
@@ -44,6 +44,10 @@ export const configSchema = {
         type: 'boolean',
         default: false
       },
+      enableLayout: {
+        type: 'boolean',
+        default: false
```
Comment (itsjwala) [inline packages/core/src/config.js:49]:
```suggestion
```

lets not add default for now. we can interpret it as false when nothing is sent.

File: packages/dom/README.md (hunk 1 above)
Comment (itsjwala) [reply packages/dom/README.md:41]: This documentation won't be part of `dom` package, as we don't use it here.

Comment (prklm10) [reply packages/dom/README.md:41]: done

File: packages/core/src/config.js (hunk 2 above)
Comment (prklm10) [reply packages/core/src/config.js]: removed default value.

**Review by itsjwala — APPROVED** [review]

**Pull request merged**